content/en/docs/installation/supported-releases.md

```diff
@@ -230,18 +230,43 @@ Our testing coverage is:
 The oldest Kubernetes release supported by cert-manager is 1.16, as we want
 to be supporting most commercial Kubernetes offerings.
 
-|   Vendor   | Oldest Kubernetes Release\* |                Other Old Kubernetes Releases                 |
-|:----------:|:---------------------------:|--------------------------------------------------------------|
+|   Vendor   | Oldest Kubernetes Release\* | Other Old Kubernetes Releases                                |
+| :--------: | :-------------------------: | ------------------------------------------------------------ |
 | [EKS][eks] |     1.16 (EOL Jul 2021)     | 1.17 (EOL Sep 2021), 1.18 (EOL Nov 2021), 1.9 (EOF Apr 2022) |
 | [GKE][gke] |     1.17 (EOL Nov 2021)     | 1.18 (EOL Dec 2021), 1.19 (EOL Feb 2022)                     |
 | [AKS][aks] |     1.18 (EOL Jul 2021)     | 1.19 (EOL Aug 2021)                                          |
 
-\*As of June 15, 2021.
+\*As of June 22, 2021.
 
 [eks]: https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html#kubernetes-release-calendar
 [gke]: https://cloud.google.com/kubernetes-engine/docs/release-schedule
 [aks]: https://docs.microsoft.com/en-us/azure/aks/supported-kubernetes-versions#aks-kubernetes-release-calendar
 
+## OpenShift supported versions {#openshift-supported-versions}
+
+With regards to OpenShift Container Platform 3, cert-manager 1.2 is the last
```

Comment:
nitpick: I wasn't sure about this off the top of my head, but from googling it seems the correct grammar would be "With regard to".

Reply:
You are totally correct.

On a different note, Grammarly is giving me a warning about "with regard to" 😅

> ## Inflated phrase
> 
> The phrase With regards to may be wordy. Consider changing the wording.
>
> Using a long phrase when a shorter one (or even a single word) will suffice can contribute to wordiness or vagueness. Though a sentence may be grammatically correct, writing more concisely is often a better choice. Consider your reader and context to make a determination.